apps/api/src/routes/SmsRoutes.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import { StatusCodes } from 'http-status-codes';
+import kafka from '../loaders/kafka';
+import { SmsService } from '../services/SmsService';
+
+const smsRoutes = Router();
+
+smsRoutes.post('/', async (req, res, next) => {
+    try {
+        const smsService = new SmsService(kafka);
+
+        const newSms = req.body;
+
+        smsService.send(newSms);
+
+        res.json({ message: 'SMS sent'});
+    } catch (error) {
+        next(error);
+    }
+});
```

Comment:
SMS should be triggered by the MessageService to prevent spam, and unintentional triggers.